### PR TITLE
Feat/pci project creation

### PR DESCRIPTION
**Pull request by Jisay**

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/new/description/description.html [hunk 1]
```diff
@@ -1,5 +1,7 @@
 <form class="pci-projects-new-description"
-      name="projectDescriptionForm">
+      name="projectDescriptionForm"
+      id="projectDescriptionForm"
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/projects/new/description/description.html]: Is is used ?

File: packages/manager/modules/pci/src/projects/new/new.controller.js [hunk 2]
```diff
@@ -137,6 +136,9 @@ export default class PciProjectNewCtrl {
 
     if (hasCredit) {
       createParams.credit = this.paymentModel.credit.value;
+    } else if (this.newProjectInfo.order
+      && (!this.paymentStatus || ['success', 'accepted'].indexOf(this.paymentStatus) > -1)) {
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/projects/new/new.controller.js]:
```suggestion
      && (!this.paymentStatus || ['success', 'accepted'].includes(this.paymentStatus))) {
```

File: packages/manager/modules/pci/src/projects/new/new.html [hunk 3]
```diff
@@ -38,29 +39,24 @@ <h1 data-translate="pci_projects_new_title">
...
-                            data-variant-nav="next"
-                            data-on-click="$ctrl.onNextBtnClick()">
-                            <span data-ng-bind="$ctrl.getNextButtonText()"></span>
-                        </oui-button>
-
-                        <a class="oui-button oui-button_primary oui-button_icon-right"
-                        data-ng-href="{{ $ctrl.getNextLinkHref() }}"
-                        data-ng-if="!$ctrl.isNextButtonVisible()">
+                        <button class="oui-button oui-button_primary oui-button_icon-right"
+                                type="submit" form="projectDescriptionForm"
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/projects/new/new.html]: Ah, yes it is used :D

File: packages/manager/modules/pci/src/projects/new/new.html [hunk 4]
```diff
@@ -38,29 +39,24 @@ <h1 data-translate="pci_projects_new_title">
...
+                        </button>
 
                         <a data-ng-href="{{ $ctrl.getStateLink('cancel') }}"
-                        data-translate="pci_projects_new_cancel">
+                           data-translate="pci_projects_new_cancel">
                         </a>
                     </div>
+
+                    <p data-ng-if="$ctrl.isCreditAsteriskVisible()">
+                        <small data-translate="pci_projects_new_create_credit_asterisk"></small>
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/projects/new/new.html]: `small` should be used only for side-comments and small prints (copyright notices and legal texts)

File: packages/manager/modules/pci/src/projects/new/new.routing.js [hunk 5]
```diff
@@ -64,21 +64,29 @@ export default /* @ngInject */ ($stateProvider) => {
...
-        //       return null;
-        //     }
-        //     return $q.reject(error);
-        //   }),
+        hasCreditToOrder: /* @ngInject */ (getCurrentStep, newProjectInfo, paymentStatus) => () => {
+          const currentStep = getCurrentStep();
+
+          return newProjectInfo.order
+            && ((!paymentStatus && currentStep.model.defaultPaymentMethod)
+            || ['success', 'accepted'].indexOf(paymentStatus) > -1
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/projects/new/new.routing.js]:
```suggestion
            || ['success', 'accepted'].includes(paymentStatus)
```

File: packages/manager/modules/pci/src/projects/new/payment/creditInfo/creditInfo.html [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+    <oui-message
+        data-type="info"
+        data-dismissable="false">
+        <span data-translate="pci_projects_new_payment_credit_info_message"
+              data-translate-values="{ amout: $ctrl.amountPrice.text }">
+        </span>
+    </oui-message>
+
+    <p>
+        <small data-translate="pci_projects_new_payment_credit_info_explain"
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/projects/new/payment/creditInfo/creditInfo.html]: Small story, `small` has a semantic meaning

File: packages/manager/modules/pci/src/projects/new/payment/payment.html [hunk 7]
```diff
@@ -5,126 +5,148 @@
...
-         data-ng-switch="$ctrl.step.model.mode">
+    <div data-ng-if="!$ctrl.step.loading.init">
 
-        <div data-ng-switch-when="paymentMethod">
-            <pci-project-new-payment-default
-                data-ng-if="$ctrl.step.model.defaultPaymentMethod"
-                data-default-payment-method="$ctrl.step.model.defaultPaymentMethod"
-                data-payment-method-url="$ctrl.paymentMethodUrl">
-            </pci-project-new-payment-default>
+        <div data-ng-if="['success', 'accepted'].indexOf($ctrl.paymentStatus) === -1"
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/projects/new/payment/payment.html]:
```suggestion
        <div data-ng-if="!['success', 'accepted'].includes($ctrl.paymentStatus)"
```

File: packages/manager/modules/pci/src/projects/new/payment/payment.html [hunk 8]
```diff
@@ -5,126 +5,148 @@
...
+                        data-translate="pci_project_new_payment_dlp_amount"
+                        data-translate-values="{ amount: $ctrl.dlpStatus.balance.text }">
+                </strong>
+                <small data-translate="pci_project_new_payment_valid_voucher_explain"></small>
+            </div>
+
         </div>
 
-        <div data-ng-if="$ctrl.dlpStatus">
+        <div data-ng-if="['success', 'accepted'].indexOf($ctrl.paymentStatus) > -1 && $ctrl.newProjectInfo.order">
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/projects/new/payment/payment.html]:
```suggestion
        <div data-ng-if="['success', 'accepted'].includes($ctrl.paymentStatus) && $ctrl.newProjectInfo.order">
```

**Comment by FredericEspiau** [comment]
It's a good work, I think you should change the `small` to `p` with a smaller font and replace `indexOf` as it's really not convenient :D

**Comment by antleblanc** [comment]
Please target `develop` branch instead:)

**Pull request merged**